src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -181,7 +183,8 @@ internal async Task<(bool, SumType<CompletionItem[], CompletionList>?)> TryGetPr
                 TextSpan.FromBounds(previousCharacterProjection.PositionIndex, previousCharacterProjection.PositionIndex + 1),
                 string.Empty);
 
-            trackingDocumentManager.UpdateVirtualDocument<CSharpVirtualDocument>(documentSnapshot.Uri, new[] { removeProvisionalDot }, previousCharacterProjection.HostDocumentVersion);
+            textChanges = (new[] { removeProvisionalDot }).ToVisualStudioTextChangeList();
```

Comment:
ditto q

Reply:
Agreed, no good reason, not thinking I guess :) 

---
In reply to: [447116029](https://github.com/dotnet/aspnetcore-tooling/pull/2137#discussion_r447116029) [](ancestors = 447116029)